flow-typed/npm/babel-cli_vx.x.x.js

```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: 8615cf6c39596ff7ba27c4a0377fe299
```

Comment:
Is it _really_ necessary to version all of these auto-generated files? Why do we even care about `babel-cli`'s Flow annotations?